snowflake/DataIngestWithCopyTask.txt

```diff
@@ -0,0 +1,33 @@
+-- Select database -- 
+
+use database TEST_DB;
+
+-- Create stage -- 
+
+create or replace stage azure_copy
+  URL='azure://<storage-account>.blob.core.windows.net/<blob>'
+  CREDENTIALS=(AZURE_SAS_TOKEN='<sas-token>');
+  
+list @azure_copy;
+
+-- Create task -- 
+create or replace task copy_data
+WAREHOUSE=PLAYWH
```

Comment:
There are still some places to unify letters case. What is a convention? Right now we have `create or replace` but `WAREHOUSE=PLAYWH`